modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
-|`${threshold}` |Count over timeframe which triggers the alert |Conditional
+|`${priority}` |Priority of the notification triggering the alert |Custom Application Notification
+|`${resource}` | Name of the resource triggering the alert, such as server or app | All
+|`${severity}` | Severity of the alert |All
+|`${state}` | State of the resource triggering the alert, such as Running or Disconnected| All
+|`${threshold}` |Count over the timeframe which triggers the alert |Conditional
 |`${value}` | A value associated with an element | Conditional
 
 |===
 
-== Editing Existing Alerts
-
-To see the alerts for an application or server, click the *View Alerts* link in the panel for that application or server. You can also click the *Alerts* link in the left panel to view a list of all alerts.
-
-image::view-alerts.png[alerts]
+== Access Alerts
 
+You can access alerts in Runtime Manager in several ways.
 
-From this menu you can filter the displayed alerts, click on an alert name to directly edit it, or click the icon next to one to open its panel and view its settings. From this panel you can click the *edit* icon next to the name to edit the same options you have when creating a new alert.
+To display all the alerts associated with your account, click *Alerts* in the left navigation pane:
 
-image::edit-alert.png[edit]
+.The arrow shows the *Alerts* menu.
+image::alerts-view-all-alerts-tab.png[Alerts menu]
 
+From this page, you can:
 
+* Create an alert.
+* Search for a string in an alert name.
+* Filter the displayed alerts by source, condition, severity, or status.
+* Click an alert name to change its settings.
+* Select an alert to display its status, settings, and history in the details pane:
++
+.The arrow shows the alert history in the details pane.
+image::alerts-history.png[Alert history in the details pane]
 
-=== Switching an Alert On or Off
+To display the alerts associated with an application or server:
 
-To change an alert's state, select one or more alerts and click the switch to *on* or *off*.
+. In the *Applications* or *Servers* list, click the *Status* column for the application or server.
+. Click the *View Associated Alerts* link in the details pane:
++
+.The arrow shows the *View Associated Alerts* link in the details pane.
+image::alerts-view-details-pane.png[View Associated Alerts link in the details pane]
 
-image::alert-on.png[on]
 
-All users of Anypoint Platform, even those without permissions to create alerts, can control whether email alerts are sent by setting each alert to an *Active* or *Inactive* state.
+To display the alert history for an application or server:
 
-[NOTE]
-When an alert is disabled or enabled the change is applied for all users who receive email notifications for that alert.
+. In the *Applications* or *Servers* list, click the application or server name.
+. Click *Settings* in the left navigation pane.
+. Click the *Alerts History* tab:
```

Comment:
I believe Alerts History tab only shows for applications deployed to local servers if we want to clarify that.